lib/crasher.js

```diff
@@ -33,19 +33,35 @@ function logCrashAndExit (logPath, errorInfo, checkout, stackTrace, err) {
   }
 }
 
-function setupCrashHandlers (processName, swap) {
+function printAndExit (errorInfo, stackTrace, err) {
+  // Electron does not play well with process.exit, so use app.exit instead
+  const runContext = isBare ? global.Bare : (global.process.versions.electron ? require('electron').app : global.process)
+  const pid = isBare ? Bare.pid : global.process.pid
+  const errInfo = err !== null && typeof err === 'object' ? JSON.stringify(err, 0, 4).slice(1, -2) : ''
+  const errorMsg = `${stackTrace + errInfo}\n\n${errorInfo}`
+
+  console.error(errorMsg)
+
+  if (isBare) {
```

Comment:
imo, this part should happen outside the function so we dont have to duplicate

Reply:
updated